packages/core/test/Form_test.js

```diff
@@ -1419,22 +1468,6 @@ describeRepeated("Form common", createFormComponent => {
         minLength: 8,
       };
 
-      it("should update the errorSchema on form submission", () => {
```

Comment:
Why did you remove this test? Is there already another test that makes this one redundant?

Reply:
yes, just below this one